contracts/truefi/TrueFiPool.sol

```diff
@@ -257,6 +265,64 @@ contract TrueFiPool is ITrueFiPool, ERC20, ReentrancyGuard, Ownable {
         emit Exited(msg.sender, amount);
     }
 
+    /**
+     * @dev Exit pool only with liquid tokens
+     * This function will withdraw TUSD but with a small penalty
+     * @param amount amount of pool tokens to redeem for underlying tokens
+     */
+    function liquidExit(uint256 amount) external nonReentrant {
+        require(amount <= balanceOf(msg.sender), "CurvePool: insufficient funds");
+
+        uint256 amountToWithdraw = poolValue().mul(amount).div(totalSupply());
```

Comment:
Maybe `valueToWithdraw`?

Reply:
why? amount seems to suit better

Reply:
It was a bit confusing at the beginning, but maybe you're right